ProjectGroupA/src/main/java/com/fundation/search/View/MainContainer.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  @(#)AssetFile.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *  <p>
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ */
+
+package com.fundation.search.View;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.Map;
+
+public class MainContainer extends Container {
+    private ResultsPanel resultsPanel;
+    private ParametersPanel paramnetersPanel;
+
+    public MainContainer(){
```

Comment:
Need to keep a consistency with the rest it should be:
public MainContainer() {
Please notice the space before the {
The same goes for the rest of the methods, check Controller code please.

Reply:
to be fixed in next commit